### ISSUE TITLE
Assembling reads before diamond

### ISSUE THREAD

**Issue by eousley** [bbuchfink/diamond#565]
Hi, I am currently trying to use diamond on my 250bp paired end reads separately to the use the daa2rma tool from MEGAN to combine the output files and run MEGAN. I am working with 128 cores on 512 GBs. I have also tried to -b5 -c1 -k1 to speed up Diamond, however it is still taking over 6 hours to run. I was wondering if assembling my paired end reads together using something like Megahit will help Diamond run more efficiently? 

My current code: $diamond blastx -d nrd.dmnd -q Read.R1.fastq -o Read.R1.daa -b5 -c1 -k1

I am using the NCBI database.

Thanks,
Emalee

**Comment by bbuchfink** [comment]
Yes assembly will usually reduce the amount of data by a pretty big factor.

**Comment by eousley** [comment]
Thank you for you response! It still took about 4 hours but it worked!
